scenarios/kubernetes_janitor.py

```diff
@@ -106,6 +106,7 @@ def check_ci_jobs():
     match_re = re.compile(r'--gcp-project=(.+)')
     for value in config.values():
         clean_hours = 24
```

Comment:
If `--soak` will always be a separate element in the list, it'd be cleaner IMO to set the hours before entering the for loop.
```python
        clean_hours = 24
        args = value.get('args', [])
        if '--soak' in args:                    
            clean_hours = 24 * 10
        for arg in args:
            ....
```